front/src/applications/editor/tools/rangeEdition/catenary/CatenaryEditionLayers.tsx

```diff
@@ -210,14 +210,22 @@ export const CatenaryEditionLayers: FC = () => {
   );
 };
 
-export const CatenaryMessages: FC = () => null;
-
-// export const CatenaryMessages: FC = () => {
-//   const { t } = useTranslation();
-//   const {
-//     state: {
-//       /* TODO */
-//     },
-//   } = useContext(EditorContext) as ExtendedEditorContextType<SpeedSectionEditionState>;
-//   return null;
-// };
+export const CatenaryMessages: FC = () => {
+  const { t } = useTranslation();
+  const { state } = useContext(EditorContext) as ExtendedEditorContextType<
+    RangeEditionState<CatenaryEntity>
+  >;
+  if (state.hoveredItem && state.hoveredItem.type === 'TrackSection')
+    return t('Editor.tools.catenary-edition.help.add-track', {
+      track: state.hoveredItem,
+      voltage: state.initialEntity.properties.voltage,
+    });
+
+  if (state.hoveredItem && state.hoveredItem.speedSectionItemType === 'TrackRange')
+    return t('Editor.tools.catenary-edition.help.remove-range', {
+      range: state.hoveredItem,
+      voltage: state.initialEntity.properties.voltage,
+    });
+
+  return t('Editor.tools.catenary-edition.help.init');
+};
```

Comment:
I think we already had this discussion a few month ago but I find it very hard to see the help message at the very bottom of the screen. Wouldn't it be better to display it on the left section where we already have the same message as the "init" one ?

Reply:
I'm agree to say that this "hint bar" is not enough visible, but its content is really not the same that the content we can see on the left panel. . 
Mainly all the hints in the bar are related to the map, not the form, that's why it's at the bottom of the map. 

There is a UX  issue here,but we've a designer now. For the v1, it will be like this